src/createSession.js

```diff
@@ -1,3 +1,35 @@
+import axios from 'axios';
+
+const libraryPackage = require('../package.json');
```

Comment:
Some suggestions:
- Rename libraryPackage to 'pkg' 
- Using process.env.npm_package_version should work as well.

Reply:
I don't know if I like `pkg` as a name because it's not as descriptive. Thanks for the suggestion of using `process.env.npm_package_version`, I have not used that before.